change/@fluentui-react-timepicker-compat-c6851c17-a774-4294-a544-72e6f554f6ee.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: release stable",
```

Comment:
this might be rather confusing for users. from semver POV this is not true 
'stable === 0.1.0" ( which you're saying by this change).

I'd suggest to keep the change message just `feat: release` or something like that.